Test/CoreSDK.Test/Operation.AL.Shared.Tests/TelemetryClientExtensionAsyncTests.cs

```diff
@@ -86,23 +88,20 @@ public void ContextPropogatesThruAsyncAwait()
         /// Ensure that context being propagated via Begin/End.
         /// </summary>
         [TestMethod]
-        public void ContextPropogatesThruBeginEnd()
+        public void ContextPropagatesThruBeginEnd()
         {
             var op = this.telemetryClient.StartOperation<RequestTelemetry>("request");
             var id1 = Thread.CurrentThread.ManagedThreadId;
             int id2 = 0;
             this.telemetryClient.TrackTrace("trace1");
 
-            HttpWebRequest request = WebRequest.Create(new Uri("http://bing.com")) as HttpWebRequest;
-            var result = request.BeginGetResponse(
+            var result = Task.Delay(millisecondsDelay: 50).AsAsyncResult(
```

Comment:
There is an additional Thread.Sleep(100); call in this method that seems questionable.

I also wonder if the test method itself should be attributed for timeout so that the while (!result.IsCompleted) loop doesn't ever accidentally become an infinite loop.